src/components/Layout/Layout.module.scss

```diff
@@ -21,3 +23,22 @@
 .active {
   color: #E30464;
 }
+
+.rdsHeader{
+  display: flex;  
+  align-self: flex-start; 
+  
+}
+
+.logo {
+  width: 3rem;
+  padding: 0.1rem;
+}
+
+ .heading {
+  font-size: 1.5rem;
```

Comment:
What is the font size of the root?

Reply:
html {
  font-size: 15px;
  margin: 0 2em;
}

Reply:
@ankushdharkar , I see few checks being failed (2/5), Has my changes causing this concern?

Reply:
No @Harshi7016,
Those checks were failing because of my deploy.